src-java/floodlight-service/floodlight-modules/src/test/java/org/openkilda/floodlight/service/FeatureDetectorServiceTest.java

```diff
@@ -139,7 +140,7 @@ public void eswitch500Software() {
         discoveryCheck(makeSwitchMock("NoviFlow Inc", "NW500.0.1", "WB5164-E", OFVersion.OF_13, 2),
                 ImmutableSet.of(
                         GROUP_PACKET_OUT_CONTROLLER, BFD, METERS, RESET_COUNTS_FLAG, INACCURATE_METER, MATCH_UDP_PORT,
-                        MULTI_TABLE, NOVIFLOW_PUSH_POP_VXLAN));
+                        MULTI_TABLE, NOVIFLOW_PUSH_POP_VXLAN, HALF_SIZE_METADATA));
```

Comment:
this class contains 2 other tests with WB switches (line 113, line 162). But those tests are green. seems method `is100GbHw` works wrong

Reply:
I have verified `is100GbHw` check and it works incorrectly. I will make fix for it in separate PR.

Reply:
Looks like `isNoviSwitch` works incorrect as well (as you had to add E switch check in `HALF_SIZE_METADATA` feature detector). But the probkem is not related to this patch. Will fix in separate PR.